src/core/crypto/Crypto.ts

```diff
@@ -171,7 +177,7 @@ export class Crypto {
         try {
             const decoded = Crypto._decode(recipientPrivate, senderPublic, payloadBuffer, tagAndIv);
             return decoded.toUpperCase();
-        } catch {
+        } catch (e) {
             // To return empty string rather than error throwing if authentication failed
```

Comment:
What was the reasoning behind returning an empty string rather than an error?

Reply:
You are right, we should probably raise an Error. This is a behaviour change, we could do it now too

Reply:
Updated to raise an error. Note that clients would need to catch them if they aren't decrypting correctly.